subgraph-js/src/directives.ts

```diff
@@ -110,23 +110,13 @@ export const federationDirectives = [
   ProvidesDirective,
   ShareableDirective,
   LinkDirective,
+  TagDirective,
 ];
 
 export function isFederationDirective(directive: GraphQLDirective): boolean {
   return federationDirectives.some(({ name }) => name === directive.name);
 }
 
-export const otherKnownDirectives = [TagDirective];
```

Comment:
Quick note for reviewers: an option could have been to keep that notion of "known but not officially federation" directives but have it empty for now. But if anything, we're trying to move away from hard-coded directives and toward `@link`-defined directives, so I'm almost certain this would never be used again, so removed it (in fact, https://github.com/apollographql/federation/pull/1554 is probably going to change all of this before GA anyway).